Formula/scorecard.rb

```diff
@@ -0,0 +1,31 @@
+class Scorecard < Formula
+  desc "Security health metrics for Open Source"
+  homepage "https://github.com/ossf/scorecard"
+  url "https://github.com/ossf/scorecard/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "9d3f0dd0a82cdc4cb3cd69149ad0967291b1bafb751bb07bc9c92db00af613a9"
+  license "Apache-2.0"
+  head "https://github.com/ossf/scorecard.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
+
+    system "make", "install"
+    system "make", "build-scorecard"
+    bin.install "scorecard"
```

Comment:
What does it `install` if that is before  the `build`, and why do you still need the `bin.install` after?

Reply:
It looks like `make install` isn't necessary so I'll remove it. 👍🏻

I'd normally have just used `make build`, but that seems to try and install it with a Docker environment. So I've just used the `build-scorecard` which doesn't actually install it into the system (hence why the `bin.install` is needed to add it to the path).